bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus_run_card.dat

```diff
@@ -0,0 +1,161 @@
+#***********************************************************************
+#                        MadGraph5_aMC@NLO                             *
+#                                                                      *
+#                      run_card.dat aMC@NLO                            *
+#                                                                      *
+#  This file is used to set the parameters of the run.                 *
+#                                                                      *
+#  Some notation/conventions:                                          *
+#                                                                      *
+#   Lines starting with a hash (#) are info or comments                *
+#                                                                      *
+#   mind the format:   value    = variable     ! comment               *
+#                                                                      *
+#   Some of the values of variables can be list. These can either be   *
+#   comma or space separated.                                          *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+ 150 = nevents ! Number of unweighted events requested 
```

Comment:
Suggest you to move this up to 2K-3K for pilot run

Reply:
What would it change? Again I just cloned the 2016 configuration: https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WZTolnujj_01j_aTGC_semilep_Zpt_WVmass_5f_NLO_FXFX/WZTolnujj_01j_aTGC_semilep_Zpt-180toInf_WVmass-800toInf_5f_NLO_FXFX/WZTolnujj_01j_aTGC_semilep_Zpt-180toInf_WVmass-800toInf_5f_NLO_FXFX_run_card.dat#L31-L34

Reply:
It's just to make sure the settings you have don't crash during pilot run. Like, if the number is too small, some phase space might crash but might not be spotted during gridpack generation. But when you produce actual samples it might happen. So this is to avoid such situations, just being conservative/safe

Reply:
Ok, I did not get any crash by the way. Here you can find a log file as example: ~fernanpe/public/for_Sihyun/WWTolnulnu_01j_aTGC_lep_WWmass-0to400_4f_NLO_FXFX_W_plus.log

Reply:
Ok since you already have the gridpacks, if the rest of the things are fine for Alexander as well this nevents doesn't really need to be fixed I guess.